### PR TITLE
[SDK-3702] README redesign

### PR DESCRIPTION
**Pull request by poovamraj**
### Changes
- Changed to new README design
- Added code of conduct page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,30 +237,30 @@ By default you should at least use the following files:
...
+---
 
-This project is licensed under the MIT license. See the [LICENSE](LICENSE.md) file for more info.
+<p align="center">
+  <picture>
+    <source media="(prefers-color-scheme: light)" srcset="./assets/auth0_brand_light.png"   width="150">
+    <source media="(prefers-color-scheme: dark)" srcset="./assets/auth0_brand_dark.png" width="150">
+    <img alt="Auth0 Logo" src="./assets/auth0_brand_light.png" width="150">
+  </picture>
+</p>
```
Comment (stevehobbsdev) [inline README.md:263]: These can now be added from the CDN and the local files deleted, see https://auth0.slack.com/archives/C0414QJGXG9/p1666171760950799?thread_ts=1666170245.671199&cid=C0414QJGXG9

File: README.md [hunk 2]
```diff
@@ -245,30 +237,30 @@ By default you should at least use the following files:
...
-1. Go to [Auth0](https://auth0.com) and click Sign Up.
-2. Use Google, GitHub or Microsoft Account to login.
+### Contributing
 
-## Issue Reporting
+We appreciate feedback and contribution to this repo! Before you get started, please see the following:
 
-If you have found a bug or if you have a feature request, please report them at this repository issues section. Please do not report security vulnerabilities on the public GitHub issue tracker. The [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues.
+- [Auth0's general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)
+- [Auth0's code of conduct guidelines](https://github.com/auth0/Lock.Android/blob/main/CODE-OF-CONDUCT.md)
```
Comment (stevehobbsdev) [inline README.md:247]: Could this just be a relative link?

**Review by stevehobbsdev — DISMISSED** [review]
LGTM, just left a couple of things

File: README.md [hunk 3]
```diff
@@ -1,27 +1,20 @@
...
 
-* **Integrates** your Android app with **Auth0**.
-* Provides a **beautiful native UI** to log your users in.
-* Provides support for **Social Providers** (Facebook, Twitter, etc.), **Enterprise Providers** (AD, LDAP, etc.) and **Username & Password**.
-* Provides Passwordless authentication using **SMS** and **Email**.
+## Documentation
 
-## Users from Lock.Android v2
-If you are migrating from the previous major version of this library, make sure to check out the [Migration Guide](MIGRATION_GUIDE.md) we prepared for you.
+- [Examples](https://github.com/auth0/auth0.android/blob/main/EXAMPLES.md)
```
Comment (frederikprijck) [inline README.md:11]:
```suggestion
- [Examples](https://github.com/auth0/Lock.android/blob/main/EXAMPLES.md)
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**